projects/super_res/config.py

```diff
@@ -9,13 +9,13 @@
 config.random_fourier_features = True,
 config.learned_sinusoidal_dim = 32
 config.diffusion_steps = 1500
-config.sampling_steps = 6
-config.loss = "l1"
+config.sampling_steps = 20
+config.loss = "l2"
```

Comment:
This seems like a major change. It would be great if there was a clearly defined experiment behind this choice.

Reply:
I ran a couple of experiments with less number of epochs. But it was cluttering the project space of wandb so I deleted them